rasa/nlu/test.py

```diff
@@ -146,24 +148,38 @@ def log_evaluation_table(
 
 
 def get_evaluation_metrics(
-    targets: Iterable[Any], predictions: Iterable[Any], output_dict: bool = False
+    targets: Iterable[Any],
+    predictions: Iterable[Any],
+    output_dict: bool = False,
+    include_no_entity: bool = True,
```

Comment:
shall we use the same default everywhere?

Reply:
The function is also used to evaluate intents. That is why I added the default value.

Reply:
I see, maybe invert and rename `include_no_entity` to smth like `exclude_label: Text`?